resources/assets/js/views/FSL/cash_logger/cash_logger.vue

```diff
@@ -0,0 +1,415 @@
+    import {mapGetters} from "vuex";
+    import BasePagination from '../../../components/Pagination/BasePagination';
+    import queryParams from '../../../utilities/queryParam';
+
+
+    export default {
+        name: 'cash_logger',
+        components: {
+            AutoCompleteSearch,
+            Orders,
+            CustomerProfile,
+            CustomHeader,
+            CustomSMSButton,
+            BasePagination
+        },
+        data() {
+            return {
+                user: {
+                    name: Auth.state.user_name,
+                    id: Auth.state.user_id
+                },
+                error: {},
+                paymentMethod: {},
+                paymentType: {},
+                activeItem: 'home',
+                paymentHeaders: ["Index", "Transaction Id", "Date of Payment", "Payment Purpose", "Payment Type", "Amount Paid", "Comment"],
+                cashLogForm: {},
+                show: false,
+                perPage: 10,
+                currentPage:1,
+                query: {
+                    page: 1,
+                    column: 'id',
+                    direction: 'desc',
+                    per_page: 10,
+                    search_column: 'id',
+                    search_operator: 'greater_than',
+                    search_input: 0
+                },
+                pageCount: 0,
+                listToOrder: 'Current',
+                customer: null,
+                customer_id: '',
+                headers: ['Date', 'Order No.', 'Product Name', 'Total Product Price',
+                    'Percentage', 'Down Payment', 'Repayment Plans'],
+                transactions: []
+
+            }
+        },
+
+        methods: {
+            async updateView(data) {
+                let {customer, user} = data;
+                if (!!customer.length) {
+                    customer = customer[0];
+                    !customer.document && (customer.document = {id_card_url: "", passport_url: ""});
+                    this.user.branch = user.branch_id;
+                    this.customer = customer;
+                    this.customer.orders = customer.orders.map(order => {
+                        let orderWithCustomer = order;
+                        orderWithCustomer.customer = this.customer;
+                        return new OrderWithPromiseCall(orderWithCustomer, this.getAuthUserDetails.userId);
+                    });
+                    this.show = true;
+                } else Flash.setError("Customer not found.", 5000);
+                this.$LIPS(false);
+            },
+
+            submitForm() {
+                this.cashLogForm.customer_id = this.customer.id;
+                this.$validator.validateAll().then(result => {
+                    if (result) {
+                        if (this.$network()) {
+                            this.$LIPS(true);
+                            post('/api/payment', this.cashLogForm).then((res) => {
```

Comment:
Let's handle our backend calls with services or store, to have a testable component and reduce side effect